config/default.yml

```diff
@@ -1783,6 +1783,11 @@ Lint/UselessSetterCall:
   Enabled: true
   VersionAdded: '0.13'
 
+Lint/UselessTrailingComma:
+  Description: 'Checks for useless trailing commas in block arguments.'
+  Enabled: false
```

Comment:
Please add `Safe: false`.

```diff
  Enabled: false
+ Safe: false
  VersionAdded: '0.80'
```

Reply:
Since the one-argument case is not changed, is this still correct?